config/defaults.ini

```diff
@@ -266,6 +261,13 @@ branding_fbe_config = ${build:datadir}/branding/gnome-initial-setup/default/gnom
 # Environment variables to substitute in branding configuration files
 branding_subst_vars_add =
 
+[kolibri]
+app_version = 0.14.6
+app_desktop_xdg_plugin_version = 1.0.8
```

Comment:
It would be nice if we could get rid of these hardcoded versions and just use the same flatpak that is being installed in the image, but I think you have tried this in the past and hit some problems, so don't worry too much about it.